core/templates/dev/head/profile/Preferences.js

```diff
@@ -32,6 +32,13 @@ oppia.controller('Preferences', [
       });
     };
 
+    var _forceDOMRefresh = function() {
+      $scope.isClean = false;
```

Comment:
Also, maybe add a comment to the top of this function explaining why it's needed.